packages/esm-patient-registration-app/src/patient-registration/validation/patient-registration-validation.test.tsx

```diff
@@ -1,129 +1,143 @@
-    const familyNameInput = screen.getByLabelText('Family Name') as HTMLInputElement;
+    it('should allow Male as a valid gender', async () => {
+      const validFormValuesWithOtherGender = {
+        ...validFormValues,
+        gender: 'Male',
+      };
+      const validationError = await validateFormValues(validFormValuesWithOtherGender);
+      expect(validationError).toBeFalsy();
+    });
 
-    await user.click(givenNameInput);
+    it('should allow Female as a valid gender', async () => {
+      const validFormValuesWithOtherGender = {
+        ...validFormValues,
+        gender: 'Female',
+      };
+      const validationError = await validateFormValues(validFormValuesWithOtherGender);
+      expect(validationError).toBeFalsy();
+    });
 
-    fireEvent.change(givenNameInput, { target: { value: givenNameValue } });
-    fireEvent.blur(givenNameInput);
-    fireEvent.change(middleNameInput, { target: { value: middleNameValue } });
-    fireEvent.blur(middleNameInput);
-    fireEvent.change(familyNameInput, { target: { value: familyNameValue } });
-    fireEvent.blur(familyNameInput);
+    it('date of birth should not be in the future', async () => {
+      const invalidFormValues = {
+        ...validFormValues,
+        birthdate: new Date('2100-01-01'),
+      };
+      const validationError = await validateFormValues(invalidFormValues);
+      expect(validationError.errors).toContain('birthdayNotInTheFuture');
+    });
 
-    return {
-      givenNameError: screen.queryByText('Given name is required'),
-      middleNameError: screen.queryByText('Middle name is required'),
-      familyNameError: screen.queryByText('Family name is required'),
-    };
-  };
+    it('should require yearsEstimated when birthdateEstimated is true', async () => {
+      const invalidFormValues = {
+        ...validFormValues,
+        birthdateEstimated: true,
+      };
+      const validationError = await validateFormValues(invalidFormValues);
+      expect(validationError.errors).toContain('yearsEstimateRequired');
+    });
+
+    it('should throw error when monthEstimated is negative', async () => {
+      const invalidFormValues = {
+        ...validFormValues,
+        birthdateEstimated: true,
+        yearsEstimated: 0,
+        monthsEstimated: -1,
+      };
+      const validationError = await validateFormValues(invalidFormValues);
+      expect(validationError.errors).toContain('negativeMonths');
+    });
 
-  testValidName('Aaron', 'A', 'Aaronson');
-  testValidName('No', '', 'Middle Name');
-  testInvalidName('', '', '', 'Given name is required', 'givenNameError');
-  testInvalidName('', '', '', 'Family name is required', 'familyNameError');
-  testInvalidName('', 'No', 'Given Name', 'Given name is required', 'givenNameError');
-  testInvalidName('No', 'Family Name', '', 'Family name is required', 'familyNameError');
+    it('should throw error when deathDate is not in future', async () => {
```

Comment:
Actually, it should throw an error if the deathDate **is a** future date. Can you check the validator and fix it?

cc: @vasharma05

Reply:
@Piumal1999 the test and validator is fine. The issue is in the test name. I changed the date to a passed date and it was not throwing the error and test was failing due to that. I updated the test name.